src/borg/archiver/rinfo_cmd.py

```diff
@@ -52,11 +52,11 @@ def do_rinfo(self, args, repository, manifest, cache):
 
             response = repository.info()
             storage_quota = response["storage_quota"]
-            used = format_file_size(response["storage_quota_use"])
+            used = format_file_size(response["storage_quota_use"], iec=True if args.iec else False)
```

Comment:
if args.iec is a boolean, you can directly give it.

if it is not (didn't check), then give `bool(args.iec)` for the same effect as your code.